src/markdown-pages/docs/cli-command-reference/simbank-cli.md

```diff
@@ -0,0 +1,116 @@
+2019-10-21 14:24:36 INFO dev.galasa.simplatform.listener.Listener <init> Loading service: dev.galasa.simplatform.listener.TelnetServiceListener listening on port: 2023
+2019-10-21 14:24:36 INFO dev.galasa.simplatform.main.Simplatform main ... services loaded
+2019-10-21 14:24:36 INFO dev.galasa.simplatform.main.Simplatform main Starting Derby Network server....
+2019-10-21 14:24:37 INFO dev.galasa.simplatform.main.Simplatform main ... Derby Network server started on port 2027
+2019-10-21 14:24:37 INFO dev.galasa.simplatform.main.Simplatform main ... Simplatform started
+```
+
+If you are a Mac or Linux user, the messages will be almost identical.
+
+4. The SimBank process has been launched, and is listening on port *2023* for Telnet connections, on port *2080* for web services connections and on port *2027* for Derby SQL connections. Neither web services or Derby connections are explored further in this section.
+
+## Manually exploring the SimBank application
+When you launch SimBank, its banking application listens on port 2023 for incoming client Telnet connections, offering an opportunity to first connect to it manually to review and understand the (simulated) transactions it supports, before subjecting it to Galasa's provided tests.
+
+### Logging in to the simulated application
+1. With Eclipse and the *Galasa SimBank* component still running, configure your 3270 terminal emulator to access port *2023* of *localhost* (or IP address 127.0.0.1 if the *localhost* alias has not been set up) via the Telnet protocol. No SSL configuration is required.
+1. Connect to the listening Telnet service with your 3270 emulator and review the logon screen:
+
+    ![SimBank logon screen](../first-steps/simbank-logon.png)
+
+1. Ensure that the cursor is in the `Userid` field - if it is not, use the TAB key to position it:
+
+    ![TAB to the userid field](../first-steps/simbank-userid.png) 
+
+1. Enter the userid `IBMUSER`
+
+    ![Enter your userid](../first-steps/simbank-ibmuser.png) 
+
+1. Press TAB to move the cursor into the `Password` field, type the password `SYS1` and press your terminal emulator's ENTER key to logon and transfer to the SimBank main menu:
+
+    ![Banktest home screen](../first-steps/simbank-banktest.png) 
+
+> *Note:* Depending on your terminal emulator, its ENTER key may not be mapped to the physical ENTER key on your computer. For example,
+> on PCOMM, by default, the ENTER key is mapped to the host machine's right CTRL key. If you are unsure about this, review
+> your terminal emulator's documentation.
+
+6. Press PF1:
+
+    ![CICS home screen](../first-steps/simbank-cics.png) 
+
+1. Press your terminal emulator's CLEAR SCREEN key.
+1. Enter the transaction name `BANK` and press your terminal emulator's ENTER key once more to get to the SimBank main menu:
+
+    ![Main banking menu](../first-steps/simbank-mainmenu.png) 
+
+As you have been progressing through this process, Eclipse has been logging selected events to its console:
```

Comment:
Since we're not using Eclipse, this should be something like "selected events will be logged to your terminal"

Reply:
+1